packages/venia-concept/src/components/TextInput/textInput.css

```diff
@@ -0,0 +1,23 @@
+.input {
+    background: white;
+    border: 1px solid rgb(var(--venia-text-alt));
+    border-radius: 2px;
+    color: rgb(var(--venia-text));
+    display: inline-flex;
+    flex: 0 0 100%;
+    font-size: 1rem;
+    height: 2.25rem;
+    margin: 0;
+    padding: calc(0.375rem - 1px) calc(0.625rem - 1px);
+    width: 100%;
+    -webkit-appearance: none;
```

Comment:
Does this just fallback to the native input then?

Reply:
All this does is disable some default styles. It's still functionally a native input; tapping the field (on a mobile device) should still spawn a native OS-level UI for actually collecting input.